HISTORY.md

```diff
@@ -20,6 +20,7 @@
 * Fix data race bug in hash linked list memtable. With this bug, read request might temporarily miss an old record in the memtable in a race condition to the hash bucket.
 * Fix a bug that `best_efforts_recovery` may fail to open the db with mmap read.
 * Fixed a bug where blobs read during compaction would pollute the cache.
+* Fixed a bug where a data race can happens when a handle from secondary cache is `Promote()` and then both the `SetIsInSecondaryCache()` and a new `Lookup()` try to set flags.
```

Comment:
Still refers to implementation details. I would say, "Fixed a data race in LRUCache when used with a secondary_cache." (If someone wants more detail, all they have to do is `git blame HISTORY.md`, find the commit associated with this release note, and look at the details.)

Reply:
The comment is addressed in the latest commit.